vendor/phpqrcode/qrencode.php

```diff
@@ -133,20 +133,22 @@ public function getCode()
         {
             $ret;
 
-            if($this->count < $this->dataLength) {
-                $row = $this->count % $this->blocks;
-                $col = $this->count / $this->blocks;
-                if($col >= $this->rsblocks[0]->dataLength) {
-                    $row += $this->b1;
-                }
-                $ret = $this->rsblocks[$row]->data[$col];
-            } else if($this->count < $this->dataLength + $this->eccLength) {
-                $row = ($this->count - $this->dataLength) % $this->blocks;
-                $col = ($this->count - $this->dataLength) / $this->blocks;
-                $ret = $this->rsblocks[$row]->ecc[$col];
-            } else {
-                return 0;
-            }
+			if ( $this->count < $this->dataLength ) {
+				$row = $this->count % $this->blocks;
+				$col = $this->count / $this->blocks;
```

Comment:
```suggestion
				$row = (int) $this->count % $this->blocks;
				$col = (int) $this->count / $this->blocks;
```
Why not cast here?

Reply:
Is this cast necessary if we are casting them in the indexes? Especially since there is additional logic below?

Reply:
I just didn't like the casting inside the indexes, so wanted to have them earlier. Although, it feels same as row and columns should be int always but let's proceed with this one if only you can verify them with your own tests.

Otherwise, just cast them in a separate line above before using them inside the index. That feels more clear.